src/core/modules/impl/handler/JsHandlerApi.ts

```diff
@@ -1,11 +1,24 @@
+import { GQLNodeInterface } from 'legacy/gqlResult';
 import { ContractDefinition } from '../../../../core/ContractDefinition';
 import { ExecutionContext } from '../../../../core/ExecutionContext';
 import { EvalStateResult } from '../../../../core/modules/StateEvaluator';
 import { SmartWeaveGlobal } from '../../../../legacy/smartweave-global';
 import { deepCopy, timeout } from '../../../../utils/utils';
-import { InteractionData, InteractionResult } from '../HandlerExecutorFactory';
+import { ContractInteraction, InteractionData, InteractionResult } from '../HandlerExecutorFactory';
 import { AbstractContractHandler } from './AbstractContractHandler';
 
+const INIT_FUNC_NAME = '__init';
+const throwErrorWithName = (name: string, message: string) => {
+  const error = new Error(message);
+  error.name = name;
+  throw error;
+};
+enum KnownErrors {
+  ContractError = 'ContractError',
+  ConstructorError = 'ConstructorError'
+}
+const ZERO_SORT_KEY = '000000000000,0000000000000,0000000000000000000000000000000000000000000000000000000000000000';
```

Comment:
I believe we already have a const for this - https://github.com/warp-contracts/warp/blob/main/src/core/modules/impl/LexicographicalInteractionsSorter.ts#L16

Reply:
fixed